ocean_lib/models/vetest/test_ve_ocean.py

```diff
@@ -19,6 +20,9 @@
 @pytest.mark.unit
 def test1(config, factory_deployer_wallet, ocean_token, veOCEAN):
     # inspiration from df-py/util/test/veOcean/test_lock.py
+    brownie.network.gas_limit(chain.block_gas_limit)
```

Comment:
Using `gas_limit()` is only relevant for pre-1559. [ref - brownie docs on gas](https://eth-brownie.readthedocs.io/en/stable/core-gas.html).

And, ganache has 1559. [ref](https://twitter.com/trufflesuite/status/1440397961150697480).

Therefore you should be able to delete this line.

If it fails unless the line's there, we need to update our mental model I guess?

Reply:
I will delete the gas limit then, thanks.